core/securitykey.py

```diff
@@ -99,10 +99,9 @@ def startClearSKeys():
 
 
 @callDeferred
-def doClearSKeys(timeStamp, cursor):
+def doClearSKeys(timeStamp: str) -> None:
 	query = db.Query(securityKeyKindName).filter("until <", datetime.strptime(timeStamp, "%d.%m.%Y %H:%M:%S"))
 	for oldKey in query.run(100, keysOnly=True):
```

Comment:
keysOnly=True doesn't have any effect in viur-datastore, and is just thrown away by kwargs.
```suggestion
	for oldKey in query.run(100):
```